Update hpcpack-cluster-node-autogrowshrink.md

IvenBach opened the pull request:
In the  'SOA example' the default value of '200,000' carried an extra zero. Updated to show '20,000'

**Sidenote: If the calculation would result in a partial core what happens?
Using example numbers:
70,001 requests sent (70,001 - 50,000)/20,000 = 1.00005
89,999 requests sent (89,999 - 50,000)/20,000 = 1.99995
In each case would it truncate, always round up/down, round up if '>1.5+' cores?

azurecla commented:
Hi __@siriusk-MSFT__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

IvenBach committed:
Update hpcpack-cluster-node-autogrowshrink.md

In the  'SOA example' the default value of '200,000' carried an extra zero. Updated to show '20,000'
